src/test/java/jimmy/mcgymmy/model/food/FoodTest.java

```diff
@@ -28,27 +28,51 @@
     @Test
     public void constructor_nullProtein_throwsNullPointerException() {
         assertThrows(NullPointerException.class, () ->
-                new Food(VALID_FOOD_NAME, null, CARBOHYDRATE, FAT));
+            new Food(VALID_FOOD_NAME, null, CARBOHYDRATE, FAT));
     }
 
     @Test
     public void constructor_nullCarbohydrate_throwsNullPointerException() {
         assertThrows(NullPointerException.class, () ->
-                new Food(VALID_FOOD_NAME, PROTEIN, null, FAT));
+            new Food(VALID_FOOD_NAME, PROTEIN, null, FAT));
     }
 
     @Test
     public void constructor_nullFat_throwsNullPointerException() {
         assertThrows(NullPointerException.class, () ->
-                new Food(VALID_FOOD_NAME, PROTEIN, CARBOHYDRATE, null));
+            new Food(VALID_FOOD_NAME, PROTEIN, CARBOHYDRATE, null));
     }
 
     @Test
     public void constructor_invalidName_throwsIllegalArgumentException() {
         assertThrows(IllegalArgumentException.class, () ->
-                new Food(INVALID_FOOD_NAME, PROTEIN, CARBOHYDRATE, FAT));
+            new Food(INVALID_FOOD_NAME, PROTEIN, CARBOHYDRATE, FAT));
     }
 
+    @Test
+    public void toStringIsCorrect() {
+        String expected1 = "Food:test food\n"
+            + "protein: 2\n"
+            + "carbs: 3\n"
+            + "fat: 4\n";
+        assertEquals(COMPARED_FOOD.toString(), expected1);
+        String expected2 = "Food:test food2\n"
+            + "protein: 100\n"
+            + "carbs: 20\n"
+            + "fat: 10\n";
+        assertEquals(new Food("test food2", 100, 20, 10).toString(), expected2);
+    }
+
+    @Test
+    public void getCaloriesIsCorrect() {
```

Comment:
I think naming it `getCalories` is enough

Reply:
I think the test convention for other test cases are like getCalories_isCorrect or smth like that x.x